lib/ex_statsd.ex

```diff
@@ -271,6 +289,20 @@ defmodule ExStatsD do
     ] |> IO.iodata_to_binary
   end
 
+  defp packet({:event, title, text, opts}, _namespace, tags, _sample_rate) do
+    [
+      "_e",
+      "{#{title |> String.length},#{text |> String.length}}",
+      ":#{title}|#{text}",
+      opts[:priority]         && "|p:#{opts[:priority]}" || "",
+      opts[:alert_type]       && "|t:#{opts[:alert_type]}" || "",
+      opts[:source_type]      && "|s:#{opts[:source_type]}" || "",
+      opts[:aggregation_key]  && "|k:#{opts[:aggregation_key]}" || "",
+      opts[:hostname]         && "|h:#{opts[:hostname]}" || "",
+      tags |> tags_suffix
+    ] |> IO.iodata_to_binary
```

Comment:
Side note, I'm pretty sure we don't actually need to be doing the IO.iodata_to_binary function call, pretty sure :gen takes iodata.